spec/takeaway_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'takeaway'
+
+describe TakeAway do
+  let(:mockmessage) { double :message, send_message: nil }
+  subject { described_class.new(mockmessage) }
+
+  describe '#read_menu' do
+    it 'returns a display of everything on the menu (currently hardcoded)' do
+      expect(subject.read_menu).to eq({ "Sweet & Sour Chicken Balls" => 3.99,
+                                        "Chicken Chowmein" => 4.50,
+                                        "Crispy Shredded Beef" => 5.90,
+                                        "Egg Fried Rice" => 3.00,
+                                        "Fried Duck with Black Bean Sauce" => 6.40 })
+    end
+  end
+
+  describe '#select_dish' do
+    it 'adds a dish to my basket when I select it' do
+      subject.add_dish('Egg Fried Rice')
+      expect(subject.basket).to eq({ "Egg Fried Rice" => 1 })
+    end
+
+    it 'adds the same dish multiple times to my basket' do
+      subject.add_dish('Egg Fried Rice', 4)
```

Comment:
Can I update the quantity of an item? e.g. add 2 egg fried rice, then add another to order 3?

Reply:
Yes :) that was done in the next test on from this